dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -553,6 +554,10 @@ private void load(String ufsPath, long mountId, long length) {
     String fileId = new AlluxioURI(ufsPath).hash();
     ByteBuf buf = PooledDirectNioByteBuf.allocate((int) (4 * mPageSize));
     try (BlockReader fileReader = createFileReader(fileId, 0, false, options)) {
+      // cache file metadata
+      getFileInfo(ufsPath, GetStatusPOptions.newBuilder().setCommonOptions(
```

Comment:
This line assumes that dora metadata cache is always available.
@huanghua78 i remember dora will fail in the tests that uses client.FileSystem to write test files first and read then back, is it still true?
wondering in the write-then-read workloads, whether we can enable the dora metadata cache as well

Reply:
I think when metadata cache is not available, load metadata would have noop, which makes sense.

Reply:
And later on, we could pass the file status directly from the master so we don't need extra ufs call here.